### PR TITLE
Added awesome-taglist post to recipes

### PR DESCRIPTION
**Pull request by streetturtle**

### CODE REVIEW

**Comment by psychon** [comment]
Actually... I do have a question: How does one install a font? Shouldn't that be explained a bit somewhere in there?

**Comment by blueyed** [comment]
@psychon 
On modern Linux you just drop it in `~/.fonts` AFAIK.

**Comment by streetturtle** [comment]
@psychon yep, this font was generated using icomoon app: [icomoon.io/app](https://icomoon.io/app) from the svg logo, so it should be ok. And regarding the font installation I think it could be slightly different depending on a Linux distro, and leaving some explanation here could be misleading for some people. So I'd prefer to leave the explanation to google if it's ok with you :)

**Comment by psychon** [comment]
Well, fine with me.

**Comment by Elv13** [comment]
> Do I understand correctly that awesomewmfont.ttf is self-created and thus there is no copyright problem with it? 👍 then

It is also quite simple to call the code directly (without the webservice) https://stackoverflow.com/questions/22124130/import-a-sequence-of-svg-files-into-fontforge-as-glyphs-and-output-a-font-file

I used this for my `~/.mutt/rc.lua` font (work in progress).

**Comment by psychon** [comment]
So, we just merge it and wait for someone to complain about the missing "how to install a font" explanation? (And then we could e.g. link to https://wiki.archlinux.org/index.php/fonts#Manual_installation which mentions that `~/.fonts` is deprecated and `fc-cache` needs to be run...)

**Comment by blueyed** [comment]
Yes, it is `~/.local/share/fonts` by now, and the wiki says that `fc-cache` is usually not necessary, too.

I think we can include instructions to put the font into `~/.local/share/fonts` after all.

**Comment by streetturtle** [comment]
@blueyed ok, I'll update the recipe then.

**Comment by copycat-killer** [comment]
Yo, `awesomewmfont` is nice!

**Comment by actionless** [comment]
cool idea, but won't it be better to use in that font not special unicode characters but normal letters (ie `awesome`) and just set taglist font to `awesomewmfont` but tagnames to `"a", "w", "e", "s" ....`?

**Comment by actionless** [comment]
also for normal letter (`a`) you could assign an icon with outline only and for capital (`A`) — filled in

**Comment by streetturtle** [comment]
Good idea! I'll try.

**Comment by streetturtle** [comment]
I'll just need to update the article and it will be ready to merge.

**Comment by streetturtle** [comment]
I think I'm done, took almost three months :)

File: recipes/awesome-taglist.mdwn [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# Awesome taglist
+
+Here is nice-looking and super easy way to customize taglist. The idea is simple - literally write 'awesome' or 'awesomewm' (if you want to keep 9 tags) in the taglist using characters from the Awesome logo.
+
+To do it you need to install font which was generated from the svg images of the letters from the logo. Download it from [here](../awesomewm-font.ttf) and place under `~/.local/share/fonts`. Then name your tags in **rc.lua** using it. Font has two types of letters: uppercase are for the bold characters:
```
Comment (blueyed) [inline recipes/awesome-taglist.mdwn]: s/install font/install a font/
s/place under/place it under/
s/Font/The font/

**Comment by streetturtle** [comment]
Thanks @blueyed! Done.

**Comment by streetturtle** [comment]
@blueyed is there anything else I could change/improve?

**Comment by blueyed** [comment]
Thank you!

**Pull request merged**